projects/app-ziti-console/src/app/app-routing.module.ts

```diff
@@ -7,7 +23,6 @@ import {
   ZacWrapperComponent,
   IdentitiesPageComponent,
   DeactivateGuardService,
-  EdgeRoutersPageComponent
```

Comment:
I'd really prefer to see no "code changes" at all in this PR and any PRs like this, even small ones like this change

Reply:
Yep, agreed. I discussed Ed, likely from a merge conflict. Should be updated now.